module-apps/application-meditation/data/OptionsData.hpp

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2017-2023, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#pragma once
+
+#include <gui/SwitchData.hpp>
+#include <functional>
+#include <chrono>
+
+namespace
+{
+    using namespace std::chrono_literals;
+    constexpr auto defaultPreparationTime{5s};
```

Comment:
How about moving it to the corresponding .cpp? It's only used there anyway.

Reply:
Now these constants will be used to initialize with default values when `settings-getValue()` returns an empty string